pymc_experimental/tests/test_linearmodel.py

```diff
@@ -44,9 +54,10 @@ def toy_X():
 
 
 @pytest.fixture(scope="module")
-def toy_y(toy_X):
-    y = 5 * toy_X["input"] + 3
-    y = y + np.random.normal(0, 1, size=len(toy_X))
+def toy_y(toy_X, toy_actual_params):
+    y = toy_actual_params["slope"] * toy_X["input"] + toy_actual_params["intercept"]
+    np.random.seed(427)
+    y = y + np.random.normal(0, toy_actual_params["obs_error"], size=len(toy_X))
```

Comment:
Better not to mess with global seeding

```suggestion
    rng = np.random.default_rng(427)
    y = y + rng.normal(0, toy_actual_params["obs_error"], size=len(toy_X))
```